src/HttpHeader.cc

```diff
@@ -400,9 +400,11 @@ HttpHeader::parse(const char *header_start, size_t hdrLen, Http::ContentLengthIn
         const char *field_start = field_ptr;
         const char *field_end;
 
+        int seenLines = 0;
```

Comment:
Nit: I think "lines seen" sounds a bit more natural, but I do not insist on this change.

```suggestion
        int linesSeen = 0;
```

or

```suggestion
        int lines = 0;
```

Reply:
Done in 1a826d532